modMcf/src/org/aion/mcf/config/CfgApiRpc.java

```diff
@@ -171,7 +217,7 @@ String toXML() {
             xmlWriter.writeComment("boolean, enable/disable cross origin requests (browser enforced)");
             xmlWriter.writeCharacters("\r\n\t\t\t");
             xmlWriter.writeStartElement("cors-enabled");
-            xmlWriter.writeCharacters(String.valueOf(this.getCorsEnabled()));
+            xmlWriter.writeCharacters(String.valueOf(this.isCorsEnabled()));
```

Comment:
Why weren't the other configuration options added here?

Reply:
As is stated in the [RPC docs](https://github.com/ali-sharif/aion/wiki/JSON-RPC-API-Documentation), the other settings are for advanced users and putting them in the config would just give users to tweak those settings without really knowing what those settings do.

Reply:
the advanced config settings for the rpc-server will disappear after executing./aion.sh -c
Is that an expected behavior?